problems/decorate/problem.md

```diff
@@ -0,0 +1,48 @@
+Decorations are a good way to mix in commonly used functionality with your Seneca
+instance, using: `decorate(name, handler)`
+
+Where:
+- `name`: The name you wish to call the decorated function or object.
+- `handler`: The handler the decorate seneca with. This can be a function or an object.
+
+For instance here we add a `stamp` function:
+
+```
+var seneca = require('seneca')
+
+seneca.decorate('stamp', (pattern) => {
+  console.log(Date.now(), pattern)
+})
+
+seneca.stamp('role:echo')
+```
+
+Also Plugins are a good place where to add decorations. See below:
+
```

Comment:
"A further enhancement that could be done is to embed decorator in a plugin, like the code below:"